src/Web/FrontModule/Presenter/templates/SignIn.default.latte

```diff
@@ -1,9 +1,48 @@
+{templateType App\Web\FrontModule\Presenter\SignInTemplate}
```

Comment:
Can we add the "Or" divider?

<img width="567" alt="Snímek obrazovky 2024-02-05 v 11 02 41" src="https://github.com/68publishers/consent-management-platform/assets/2346295/dd4e1f9c-4184-4c0b-af7b-0a51cbc57f7f">

It can be "full button width" or shorter. It's just a Figma illustration.